File: Python Code/Daily_Flash_Code_0/Week13/14Apr/73-DailyFlash_Solutions/14-Apr_Solutionn_Two/C/prog4.c
#include<stdio.h>

int main() {
	int num = 1;
	char ch = 'A';
	for(int i = 0; i < 7; i++) {
		if(i < 4) {
			for(int j = 0; j < 4 + i; j++) {
				if(j < 3 - i)
					printf(" ");
				else {
					if(i % 2 == 0) {
						printf("%d",num);
					} else {
						printf("%c", ch);
					}

				}
			}
			num++;
			if(i % 2 == 1)
				ch++;
			if(i == 3)
				num--;
		}
		else {
			num--;
			if(i%2 == 1);
			ch--;
			for(int j = 0; j < 10 - i; j++) {
				if(j < i - 3)
					printf(" ");
				else
					if(i % 2 == 0)
						printf("%d", num);
					else
						printf("%c", ch);
			}	
		}
		printf("\n");
	}
}

File: Python Code/Daily_Flash_Code_0/Week10/25Mar/56-DailyFlash_Solutions/25-MarSolution_Three/Java/prog4.java
class Demo {
	public static void main(String[] args) {
		int num = 1;
		for(int i = 0; i < 7; i++) {
			if(i < 4) {
				for(int j = 0; j < i + 1; j++) {
					System.out.print(num);
					if(j != 3)
						num++;
				}
			} else {
				for(int j = 0; j < 7 - i; j++) {
					num--;
					System.out.print(num);
				}
			}
			System.out.println();
		}
	}
}
